index.html

```diff
@@ -7,7 +7,7 @@
     <title>Developer Dashboard | Arcana Network</title>
 
     <script>
-      window.global = window;
+      window.global = window
     </script>
```

Comment:
Just curious: Why is this needed?

Reply:
Actually it was added when we were trying to add configuration and polyfills for supporting node builtins in vite. I removed it now. It is not required. Thanks for pointing it out. It didn't came into my notice when removing unused variables and functions